core/kover/utils.py

```diff
@@ -127,4 +127,21 @@ def _unpack_binary_bytes_from_ints(a):
         b[i] = tmp > 0
         packed_rows += 1
 
-    return b
\ No newline at end of file
+    return b
+    
+def _parse_blacklist(blacklist_path):
```

Comment:
Please add a verification to that all the extracted k-mers are of the same length. Also that they are of the same length as those in the DS. You could add an expected_len param to the function.